tests/specs/lint/minified/minified.out

```diff
@@ -0,0 +1,3 @@
+File was skipped: [WILDCARD]minified.js
+   The file is minified
```

Comment:
Honestly not sure if we should print this. Thoughts?

Reply:
I personally prefer this to be printed by default. I think it could confuse some users badly if it's not printed because this type of skipping of files doesn't seem common in the ecosystem.

Reply:
Same. I'd prefer it lets me know what it's doing.